core/mondoo-aws-security.mql.yaml

```
@@ -1866,28 +1936,41 @@ queries:
     filters: |
       asset.platform == "aws-lambda-function"
     mql: |
-      aws.lambda.function.concurrency > 0
+      aws.lambda.function.concurrency <= 100
```

If we do not use properties, I suggest keeping the `> 0` check in addition to the `<=100` check, as sensible values.
```suggestion
      aws.lambda.function.concurrency > 0
      aws.lambda.function.concurrency <= 100
```